x-pack/plugins/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -67,300 +128,357 @@ export const DeploymentSetup: FC<DeploymentSetupProps> = ({
+            data-test-subj={'mlModelsStartDeploymentModalDeploymentSelectId'}
           />
-        </EuiFormRow>
-      </EuiDescribedFormGroup>
-
-      {threadsPerAllocations !== undefined ? (
-        <EuiDescribedFormGroup
-          titleSize={'xxs'}
-          title={
-            <h3>
+        )}
+      </EuiFormRow>
+
+      <EuiSpacer size="m" />
+
+      <EuiFormRow hasChildLabel={true} fullWidth>
+        <EuiFormFieldset
+          legend={{
+            children: (
               <FormattedMessage
-                id="xpack.ml.trainedModels.modelsList.startDeployment.threadsPerAllocationLabel"
-                defaultMessage="Threads per allocation"
+                id="xpack.ml.trainedModels.modelsList.startDeployment.optimizeThreadsPerAllocationLabel"
+                defaultMessage="Optimize this model deployment for your use case:"
               />
-            </h3>
-          }
-          description={
-            <FormattedMessage
-              id="xpack.ml.trainedModels.modelsList.startDeployment.threadsPerAllocationHelp"
-              defaultMessage="Increase to improve inference latency."
-            />
-          }
+            ),
+          }}
         >
+          {optimizedOptions.map((v) => {
+            return (
+              <Fragment key={v.value}>
+                <EuiCheckableCard
+                  id={v.value}
+                  disabled={isUpdate}
+                  label={
+                    <EuiText size={'s'}>
+                      <EuiFlexGroup alignItems={'baseline'} gutterSize={'s'}>
+                        <EuiFlexItem grow={false}>
+                          <strong>{v.label}</strong>
+                        </EuiFlexItem>
+                        <EuiFlexItem grow={false}>{v.description}</EuiFlexItem>
+                      </EuiFlexGroup>
+                    </EuiText>
+                  }
+                  value={v.value}
+                  checked={config.optimized === v.value}
+                  onChange={() => {
+                    onConfigChange({
+                      ...config,
+                      ...(deploymentIdUpdated.current
+                        ? {}
+                        : {
+                            deploymentId: config.deploymentId?.replace(
+                              /_[a-zA-Z]+$/,
+                              v.value === 'optimizedForIngest' ? '_ingest' : '_search'
+                            ),
+                          }),
+                      optimized: v.value,
+                    });
+                  }}
+                />
+                <EuiSpacer size="m" />
+              </Fragment>
+            );
+          })}
+        </EuiFormFieldset>
+      </EuiFormRow>
+
+      <EuiAccordion
+        id={'modelDeploymentAdvancedSettings'}
+        buttonContent={
+          <FormattedMessage
+            id="xpack.ml.trainedModels.modelsList.startDeployment.advancedSettingsLabel"
+            defaultMessage="Advanced Configurations"
```

Comment:
Should this be sentence case here `Advanced configurations` ?

Reply:
Fixed in c5f7e84cd69e880bca8c5aad5d345ff14d31c4fd